test/support/selectize_helpers.rb

```diff
@@ -2,7 +2,7 @@ module SelectizeHelpers
   def select_option(option, selector:)
     within("select#{selector}+.selectize-control") do
       first('div.selectize-input').click
-      find('div.option', text: option).click
+      find('div.option', text: option, exact_text: true).click
```

Comment:
My understarding of this was that `exact_text` should be used instead of `text`. If you want use the `text` key, then I think you can use `exact: true`.

Reply:
Ah. Yes! Thanks for this.